cdap-docs/developers-manual/source/building-blocks/streams.rst

```diff
@@ -63,12 +63,16 @@ changed, using the :ref:`http-restful-api-stream`, the :ref:`stream-client` of t
 :ref:`client-api`, or by using the :ref:`Command Line Interface. <cli>`
 
 
-.. rubric::  Truncating a Stream
+.. rubric::  Truncating and Deleting a Stream
 
 Streams can be truncated, which means deleting all events that were ever written to the
 Stream. This is permanent and cannot be undone. They can be truncated through the using
```

Comment:
I think I'm mis-reading this. (The way Github had highlighted words...) I was talking about "Truncating and Deleting a Stream"; I suspect you were talking about "...that were ever written to the Stream." Correct? 

Yes, "Stream" should be "stream", but I was going to do a big pass over all the docs and make all those changes at once. I know you have started to do this, but I was putting it off...Was that what you were thinking?